### ISSUE TITLE
TimeZone

### ISSUE THREAD

**Issue by benoj** [brianpgerson/the-count#5]
Hey,

Is it possible to set timezone for event schedules e.g. `@callie schedule: daily, event: Launch, hour: 10GMT`

And also would be great to get a list of schedules e.g. `@calie schedule, event: Launch` -> "10am GMT"

Cheers,

Ben

**Comment by brianpgerson** [comment]
That's a fun idea! If you have any interest in opening a PR, I'd be happy to review and merge it, otherwise this will go on the ol backlog til I get some time.

**Comment by boxabirds** [comment]
Yes it could be even simpler: Slack should give the timezone of the client and it should just use that. I have to set a reminder at 9am to be 0100 because I have guessed that Callie is based on the West Coast.
